File: src/main/java/com/github/knextsunj/cms/dto/StateDTO.java
package com.github.knextsunj.cms.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public record StateDTO(@JsonProperty("id") Long id, @JsonProperty("name") String name,
                       @JsonProperty("deleted") String deleted, @JsonProperty("countryId") Long countryId,
                       @JsonProperty("countryName") String countryName) implements Serializable {
}

File: src/main/kotlin/com/github/knextsunj/cms/dto/StateDTO.kt
package com.github.knextsunj.cms.dto

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

data class StateDTO(@JsonProperty("id") val  id:Long?, @JsonProperty("name") val name:String?,
                    @JsonProperty("deleted") val deleted:String?, @JsonProperty("countryId") val countryId:Long?,
                    @JsonProperty("countryName") val countryName:String?): Serializable
